suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -773,6 +773,10 @@ export const selectDeviceSupportedNetworks = (state: DeviceRootState) => {
                 return false;
             }
 
+            if (hasBitcoinOnlyFirmware(device) && !['btc', 'test', 'regtest'].includes(symbol)) {
```

Comment:
please try to reuse `['btc', 'test', 'regtest']` it is used on more places in codebase
look for `['btc', 'test', 'regtest']`, `['btc', 'test']`